standards-catalogue/docs/adding_a_standard.md

```diff
@@ -0,0 +1,35 @@
+# Adding a new standard
+
+To see a finished example of a standard, see any of the existing standards, for example [360giving](https://raw.githubusercontent.com/alphagov/data-standards-authority/adding-a-standard-docs/content/standards/360giving/index.html.md).
+
+To add a new standard in Github:
+- Go to the [standards](../../content/standards) directory
+- Create a new file called index.html.md inside a directory named after the standard. For example to create a standard for 360 Giving you'd create a file called 360giving/index.html.md.
+- Copy and paste the frontmatter template below to the top of the file and fill out the missing information:   
+
+
+```
+---
+name:  
+contraction:   
+specification_url:   
+tags:
+-
+maintainer_id:   
+licence_id:   
+endorsement_status_events:   
+- status:   
+  date: ""
+---
+```
```

Comment:
Should this instead be a `_template.md` or similar, but kept separate to make sure that it's always up to date? I can definitely see this documentation being neglected, and the next person coming along has their PR build fail despite them following docs, all because we've changed the requirements but not the docs

Reply:
Yeah, I wanted to have something like a pull request template so when they create a new standard the template is already there, but I don't think that's easily done. Will move this to another file.